packages/js/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@axiomhq/js",
     "description": "The official javascript bindings for the Axiom API",
-    "version": "1.0.0-rc.4",
```

Comment:
can probably do 1.1.0?

Reply:
but we haven't released stable `1.0.0` yet

Reply:
will release `v1.0.0` as agreed.